test/webdriverio/specs/jira.js

```diff
@@ -0,0 +1,173 @@
+describe("Jira", function () {
+
+  var bugTrackerUrl = 'https://jira.atlassian.com';
+
+  var testProjectName = 'Demo';
+  var testProjectUrl = bugTrackerUrl + '/projects/DEMO';
+
+  var testIssueName = 'Issue-qweasdzxc for ' + testProjectName;
+  var testIssueSearchUrl = testProjectUrl + '/issues?filter=reportedbyme';
+  var testIssueUrl = '';
+
+  var testFilterName = 'Filter-qweasdzxc for ' + testProjectName;
+  var testFilterSearchUrl = 'https://jira.atlassian.com/secure/ManageFilters.jspa#filterView=my';
+
+  var testBoardSearchUrl = 'https://jira.atlassian.com/secure/ManageRapidViews.jspa';
+
+  var testKanbanBoardName = 'Board-Kanban-qweasdzxc for ' + testProjectName;
+  var testKanbanBoardUrl = '';
+
+  before(function () {
+
+    var testIssueAnchorSelector = '//a[span[contains(@class,"issue-link-summary")][text()="' + testIssueName + '"]]';
+    var testKanbanBoardAnchorSelector = '//a[text()="' + testKanbanBoardName + '"]';
+
+    function createIssue () {
+      return browser
+        .url(testIssueSearchUrl)
+        .waitForExist('#create_link')
```

Comment:
Create 
client.waitAndClick(selector).then(callback);
command for shortening the scripts